bin/mozetl_runner.py

```diff
@@ -0,0 +1,17 @@
+from os import environ
```

Comment:
This part is not clear to me - is this file the same thing that's fetched from s3 here: https://github.com/mozilla/telemetry-airflow/blob/b5c78d92650ab90e07ed1969f93e76ee60ba6cf7/plugins/moz_databricks.py#L131 ?

Should the s3 script be removed and databricks operator switched to `bin/mozetl_runner.py`? (the latter seems not to be invoked anywhere right now)

Reply:
The runner in s3 is the same script as this one, I've been manually deploying it. Only `s3` and `dbfs` are supported in the [run-submit](https://docs.databricks.com/api/latest/jobs.html#jobssparkpythontask) api. I can comment the script (and move it into another folder if it makes more sense).

Reply:
For reference, there's an [open bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1484331) for deploying this dependency. However, the `mozetl` calling convention probably doesn't need to be changed for the foreseeable future.

Reply:
I see, I think adding some comment about this deployment in the script wouldn't hurt :)